### PR TITLE
Update tutorials.rst with readthedocs links

### PR DESCRIPTION
**Pull request by rdmolony**
Tutorial links linked to the old Github Pages tutorials which are still live!

### CODE REVIEW

**Comment by kvnkho** [comment]
I thought there was a redirect set on the tutorials repo but it doesn't look like it's sticking. Could you edit all of the links in the README as well to point to ReadTheDocs because we have Google Analytics enabled there?

**Comment by kvnkho** [comment]
I hope I fixed it with [this PR](https://github.com/fugue-project/tutorials/pull/131) to add a CNAME in the Jupyter Book build automatically to create the direct. I put a bit more details in the issue [here](https://github.com/fugue-project/tutorials/issues/113)

Still though, I think we wouldn't mind changing all of the links to Google Analytics for more accurate metrics tracking.

File: README.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ result.show()
 
 This syntax is simpler, cleaner, and more maintainable than the PySpark equivalent. At the same time, no edits were made to the original pandas-based function to bring it to Spark. It is still usable on pandas DataFrames. Because the Spark execution engine was used, the returned `df` is now a Spark DataFrame. Fugue `transform()` also supports `DaskExecutionEngine` and the pandas-based `NativeExecutionEngine`.
 
-## [FugueSQL](https://fugue-project.github.io/tutorials/tutorials/fugue_sql/index.html)
+## [FugueSQL](https://fugue-tutorials.readthedocs.io/tutorials/advanced/execution_engine.html)
```
Comment (kvnkho) [inline README.md:101]:
```suggestion
## [FugueSQL](https://fugue-tutorials.readthedocs.io/tutorials/fugue_sql/index.html)
```

File: README.md [hunk 2]
```diff
@@ -145,9 +145,9 @@ pip install fugue
 
 It also has the following extras:
 
-*   **sql**: to support [FugueSQL](https://fugue-project.github.io/tutorials/tutorials/fugue_sql/index.html)
-*   **spark**: to support Spark as the [ExecutionEngine](https://fugue-project.github.io/tutorials/tutorials/advanced/execution_engine.html)
-*   **dask**: to support Dask as the [ExecutionEngine](https://fugue-project.github.io/tutorials/tutorials/advanced/execution_engine.html)
+*   **sql**: to support [FugueSQL](https://fugue-tutorials.readthedocs.io/tutorials/advanced/execution_engine.html)
```
Comment (kvnkho) [inline README.md:148]:
```suggestion
*   **sql**: to support [FugueSQL](https://fugue-tutorials.readthedocs.io/tutorials/fugue_sql/index.html)
```

**Comment by kvnkho** [comment]
@rdmolony , I think two of the SQL links are off. You should be able to just commit my suggestions directly and then we can approve and merge this. Thank you!

**Comment by rdmolony** [comment]
Thanks @kvnkho for reviewing

**Review by kvnkho — APPROVED** [review]

**Pull request merged**